spec/features/user_integration_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.feature 'User Integration', type: :feature do
+  let!(:user) { create(:user) }
+
```

Comment:
- [ ] When I run the specs, The error in the image below is raised, Please fix that.

![image](https://github.com/Eccb7/Pi-Blogs/assets/68796232/30d0c626-e6d1-4f7f-95ba-cdba09ec0d19)